<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1581</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPlaylist">
     <attribute name="title">
      <string>Song Queue</string>
     </attribute>
     <widget class="QSlider" name="horizontalSliderDuration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>461</y>
        <width>621</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::handle:horizontal {
    background: white;
    border: 2px solid black;
    height: 20px;
    border-radius: 8px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStop">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>480</y>
        <width>71</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 35px;
background-color: white;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBack">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>480</y>
        <width>71</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 35px;
background-color: white;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonForward">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>480</y>
        <width>71</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 35px;
background-color: white;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPlayPause">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>480</y>
        <width>71</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 35px;
background-color: white;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonVolume">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>470</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: white;
border: 2px solid black;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetSongQueue">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>371</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSliderVolume">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>310</y>
        <width>100</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QSlider {
    min-width: 100px;
    max-width: 100px;

}

.QSlider::groove:vertical {
    border: 1px solid #262626;
    width: 10px;
    background: #393939;
    margin: 0 12px;
}

.QSlider::handle:vertical {
    background: white;
    border: 2px solid black;
    height: 10px;
    margin:  0 -10px;
	border-radius: 5px;
}
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelCurrentlyPlaying">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Currently Playing</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTrackPhoto">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>311</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEditTrackInfo">
      <property name="geometry">
       <rect>
        <x>1380</x>
        <y>450</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Artist</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRemoveSong">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>450</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClearQueue">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>450</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Queue</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetAllSongs">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>10</y>
        <width>321</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelVolume">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>270</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHistory">
     <attribute name="title">
      <string>Listening History</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetHistory">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>1031</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_Song">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSelect_File_mp3"/>
    <addaction name="actionClear_History"/>
   </widget>
   <addaction name="menuAdd_Song"/>
  </widget>
  <action name="actionSelect_File_mp3">
   <property name="text">
    <string>Select File (.mp3)</string>
   </property>
  </action>
  <action name="actionClear_History">
   <property name="text">
    <string>Clear History</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
